app/scripts/linkhandler.js

```diff
@@ -1,4 +1,3 @@
-'use strict'
 // var Router = require('react-router')
 var $ = require('jquery')
```

Comment:
yep already an issue open for that, but requires some rewriting of components, as there are very horrible things happening there with jquery..